app/src/pages/RegisterProject.svelte

```diff
@@ -33,13 +33,13 @@
   const client = getClient();
 
   const REGISTER_PROJECT = gql`
-    mutation($domain: String!, $name: String!) {
-      registerProject(domain: $domain, name: $name) {
+    mutation($project_name: String!, $org_id: String!) {
```

Comment:
The change flattening these two params from `ProjectId` and swapping their order was introduced in this PR: https://github.com/radicle-dev/radicle-registry/pull/224.

The idea is to have the api read more naturally than necessarily structurally.  "Register project 'radicle' under 'monadic'" instead of "Register project under 'monadic' with name 'radicle'".

As seen in the PR above, this was considered, discussed, and agreed on with @geigerzaehler. If after the explanation as to why this is you are still not fond of it, I recommend us giving it a go for now and see how we get used to it.